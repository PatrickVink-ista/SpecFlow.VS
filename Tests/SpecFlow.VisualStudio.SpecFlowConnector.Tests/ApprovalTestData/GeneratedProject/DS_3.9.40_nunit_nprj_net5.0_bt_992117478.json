{
    "GeneratorOptions": {
        "featureFileCount": 1,
        "scenarioPerFeatureFileCount": 1,
        "newProjectFormat": true,
        "targetFramework": "net5.0",
        "SpecFlowPackageVersion": "3.9.40"
    }
}
