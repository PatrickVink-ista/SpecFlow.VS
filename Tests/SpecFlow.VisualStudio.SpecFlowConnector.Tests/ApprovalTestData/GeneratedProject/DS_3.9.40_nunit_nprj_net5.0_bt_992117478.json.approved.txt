{
  "StepDefinitions": [
    {
      "Type": "When",
      "Regex": "^urna massa \"(.*)\" sollicitudin$",
      "Method": "LectusDignissimEuFelisSteps.WhenEgetRisusEuAccumsan(String)",
      "ParamTypes": "s",
      "Scope": null,
      "Expression": "urna massa \"(.*)\" sollicitudin",
      "Error": null,
      "SourceLocation": "#0|10|9|12|10"
    },
    {
      "Type": "Given",
      "Regex": "^ut et \"(.*)\"$",
      "Method": "LectusDignissimEuFelisSteps.GivenLoremSapienQuisqueVitae(String)",
      "ParamTypes": "s",
      "Scope": null,
      "Expression": "ut et \"(.*)\"",
      "Error": null,
      "SourceLocation": "#0|16|9|18|10"
    },
    {
      "Type": "Then",
      "Regex": "^molestie augue auctor eu$",
      "Method": "LectusDignissimEuFelisSteps.ThenTristiqueAnteLiberoUt(Table)",
      "ParamTypes": "st",
      "Scope": null,
      "Expression": "molestie augue auctor eu",
      "Error": null,
      "SourceLocation": "#0|22|9|24|10"
    }
  ],
  "SourceFiles": {
    "0": "<<targetFolder>>\\StepDefinitions\\LectusDignissimEuFelisSteps.cs"
  },
  "TypeNames": {}
}
